analyzers/tests/SonarAnalyzer.Test/TestCases/UnnecessaryBitwiseOperation.CSharp9.cs

```diff
@@ -62,4 +63,53 @@ await foreach (var v in values)
         }
         unchanged = 1;
     }
+
+    public void TupleAssignment()
+    {
+        var i = 0;
+        (i, _) = (1, 1);
+        _ = i | 0x80;
```

Comment:
Optional suggestion / Personal opinion: could we put the magic value `0x80` into a variable named `aNonSpecialValue` (or something like that), or using `42` (to which we give the meaning of "a value like any other one")?
When I first opened this test, I was trying for while to get what was special about this `0x80`, to just then figure out a few minutes later that the special thing was the other side of the operator :-)

Reply:
`0x80`, `0x010F`, and other magic numbers are used in the tests extensively. I don't think any change would help with readability. You need to load the context anyway.